src/uikit.cs

```diff
@@ -12995,7 +12998,7 @@ interface UIView : UIAppearance, UIAppearanceContainer, UIAccessibility, UIDynam
 		CoreAnimation.CALayer Layer { get; }
 
 		[Export ("frame")]
-		CGRect Frame { get; set; }
+		new CGRect Frame { get; set; }
```

Comment:
Same: can this be removed as well, so that we get it from `UIFocusItemContainer ` instead, or will the api diff complain?

Reply:
Same, protocol just exposes get, not set.